spec/abstractops.html

```diff
@@ -272,56 +288,72 @@ <h1>
+      ): either a normal completion containing either ~compatible~, ~earlier~, ~later~, or ~reject~, or a throw completion
     </h1>
     <dl class="header">
       <dt>description</dt>
       <dd>It fetches and validates the *"disambiguation"* property of _options_, returning a default if absent.</dd>
     </dl>
     <emu-alg>
-      1. If _options_ is *undefined*, return *"compatible"*.
-      1. Return ? GetOption(_options_, *"disambiguation"*, ~string~, « *"compatible"*, *"earlier"*, *"later"*, *"reject"* », *"compatible"*).
+      1. If _options_ is *undefined*, return ~compatible~.
+      1. Let _stringValue_ be ? GetOption(_options_, *"disambiguation"*, ~string~, « *"compatible"*, *"earlier"*, *"later"*, *"reject"* », *"compatible"*).
+      1. If _stringValue_ is *"compatible"*, return ~compatible~.
+      1. If _stringValue_ is *"earlier"*, return ~earlier~.
+      1. If _stringValue_ is *"later"*, return ~later~.
+      1. Return ~reject~.
     </emu-alg>
   </emu-clause>
 
   <emu-clause id="sec-temporal-getroundingmodeoption" type="abstract operation">
     <h1>
       GetRoundingModeOption (
         _options_: an Object,
-        _fallback_: a String from the "Identifier" column of <emu-xref href="#table-temporal-rounding-modes"></emu-xref>,
-      ): either a normal completion containing a String from the "Identifier" column of <emu-xref href="#table-temporal-rounding-modes"></emu-xref>, or a throw completion
+        _fallback_: a rounding mode,
+      ): either a normal completion containing a rounding mode, or a throw completion
     </h1>
     <dl class="header">
       <dt>description</dt>
       <dd>It fetches and validates the *"roundingMode"* property from _options_, returning _fallback_ as a default if absent.</dd>
     </dl>
     <emu-alg>
-      1. Let _roundingModes_ be the List of Strings from the "Identifier" column of <emu-xref href="#table-temporal-rounding-modes"></emu-xref>.
-      1. Return ? GetOption(_options_, *"roundingMode"*, ~string~, _roundingModes_, _fallback_).
+      1. Let _allowedStrings_ be the List of Strings from the "String Identifier" column of <emu-xref href="#table-temporal-rounding-modes"></emu-xref>.
+      1. Let _stringFallback_ be the value from the "String Identifier" column of the row with _fallback_ in its "Rounding Mode" column.
+      1. Let _stringValue_ be ? GetOption(_options_, *"roundingMode"*, ~string~, _allowedStrings_, _stringFallback_).
+      1. Return the value from the "Rounding Mode" column of the row with _stringValue_ in its "String Identifier" column.
     </emu-alg>
+    <p>
+      A <dfn>rounding mode</dfn> is one of the values in the "Rounding Mode" column of <emu-xref href="#table-temporal-rounding-modes"></emu-xref>.
+      An <dfn>unsigned rounding mode</dfn> is one of the values in the "Unsigned Rounding Mode" column of <emu-xref href="#table-unsigned-rounding-modes"></emu-xref>.
```

Comment:
These general terms should not be defined in a section specific to Temporal; let's finally move GetOption and GetRoundingModeOption somewhere else in ECMAScript Amendments. Maybe a new "Operations for Reading Options" subsection under [Abstract Operations](https://tc39.es/ecma262/multipage/abstract-operations.html#sec-abstract-operations)?

Reply:
It's already a bit vague where anything in https://tc39.es/proposal-temporal/#sec-temporal-abstract-ops is going to end up when the proposal is merged in, but I've added an "Operations for Reading Options" section to `spec/mainadditions.html`.